src/mutate/index.js

```diff
@@ -1,59 +1,49 @@
 import gql from "graphql-tag";
 import { graphql } from "react-apollo";
-import { compose, setDisplayName } from "recompose";
+import { compose, setDisplayName, branch, renderNothing } from "recompose";
 import _ from "underscore";
 import pluralize from "pluralize";
-import { gqlFetchDetail, gqlFetchList, mapperWrapper } from "../common";
-import withFormHandlers from "../withFormHandlers";
-import { processFormData, processSchemas } from "../utils/json-schema";
-import { schema as convertToJoi } from "enjoi";
-import joi from "joi";
+
+import { generateUISchema } from "../utils/ui-schema";
+
+import withForm from "@groundbreaker/qewl-forms";
+
+import { gqlFetchDetail, mapperWrapper } from "../common";
 
 const decorateCreateBase = args => {
-  const mutationVars = processMutationVars(args);
+  const mutationVars = processMutationVars(args, "create");
   const mutation = gqlMutate(mutationVars, args.fields);
 
-  let refetch = true;
-  if (args.refetch === false) refetch = false;
-
   return compose(
     setDisplayName(`QewlCreate(${args.resource})`),
+    withForm({ input: mutationVars.inputTypeName }),
```

Comment:
```suggestion
    setDisplayName(`Qewl(WithForm)`),
    withForm({ input: mutationVars.inputTypeName }),
    setDisplayName(`QewlCreate(${args.resource})`),
```
Will mislead on debug.